app/views/wallet/GiftingView/BuildGiftPanel/BuildGiftForm.tsx

```diff
@@ -242,7 +255,7 @@ const BuildGiftForm: FC = () => {
   const validateAmount = (selectedBalance: bigint, fee: bigint) => (valueString: string) => {
     let error;
     const valueAsPicoMob = BigInt(valueString.replace('.', ''));
-    if (valueAsPicoMob + fee > selectedBalance) {
+    if (valueAsPicoMob + fee + fee > selectedBalance) {
```

Comment:
Why `+fee+fee`? If it's to be added twice, then rather write `+2*fee`

Reply:
I was having issues with multiplying if I remember, so this was the easiest quick fix.